src/main/java/com/aws/greengrass/util/NucleusPaths.java

```diff
@@ -191,4 +197,9 @@ public static void setLoggerPath(Path p) throws IOException {
         Utils.createPaths(p);
         Permissions.setLoggerPermission(p);
     }
+
+    public Path nucleusLogsPath() {
```

Comment:
Why does this method have to be in `NucleusPaths` class?

Reply:
NucleusPaths abstracts the paths to special directories and files used by the Greengrass process. Loader log is a special file which the deployment workflow uses to track loader logs. Nucleus deployment classes already has an access to this location.